### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by zunairkhan811**
The project is completed according to the following instructions:
Add a new JavaScript file and import it as a module:
it will contain methods related to the status updates (completed: true / false).
Add event listener to the checkbox (change).
Update items object's value for completed key upon user actions.
Implement a function for the "Clear all completed" button (use filter() method).
Store the updated array of items in local storage, so the user gets the correct list of values after the page reloads (which means that any changes made to the list should be preserved).

### CODE REVIEW

**Review by abdulhamiid — APPROVED** [review]
# Status: :green_circle:  Approved! :partying_face:
Hi @zunairkhan811 👋, 

Your project is complete! Great job for nailing it right on the first attempt :1st_place_medal:. There is nothing else to say other than... it's time to merge it  :shipit:  Congratulations! :tada::100::star2:

giphy

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/clearbutton.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  const clearbtn = document.querySelector('.btn-div');
+  clearbtn.addEventListener('click', () => {
+    const newArray = JSON.parse(localStorage.getItem('Tasks'));
+
+    const filterArray = newArray.filter((e) => e.completed === false);
+    filterArray.forEach((el, i) => {
+      el.index = i;
+    });
+    localStorage.setItem('Tasks', JSON.stringify(filterArray));
+        location.reload(); // eslint-disable-line
```
Comment (fickryiman) [inline modules/clearbutton.js:11]: instead of using command to disable line eslint, suggested to use window.location.reload()

**Review by fickryiman — COMMENTED** [review]
your projects looks awesome, otherwise much better is using window.location.reload() instead of disable the linter line

**Comment by zunairkhan811** [comment]
> your projects looks awesome, otherwise much better is using window.location.reload() instead of disable the linter line

Hi! @fickryiman thanks for your suggestion. As this is the same suggestion created in the issue.
https://github.com/zunairkhan811/To-do-list/issues/6#issue-1653334483

Just wanna let you know your suggestion has been implemented 😉

**Pull request merged**